src/linagora.esn.calendar/app/settings/display/settings-display.pug

```diff
@@ -1,15 +1,21 @@
-mixin settingsSwitch(title, rightLabel, leftLabel)
-  h3 {{ title | translate }}
-  p.cal-setting-switch
-    span.working-days-disabled(ng-class="{'toggle-active': " + attributes.switchValue + "}")
-      | {{ rightLabel | translate }}
-    toggle-switch(ng-model=attributes.switchValue, color='blue', form="form")
-    span.working-days-enabled(ng-class="{'toggle-active': " + attributes.switchValue + "}")
-      | {{ leftLabel | translate }}
+//TODO: Revert to use the mixin here in case we can fix the translation issue
```

Comment:
```suggestion
// TODO: Revert to use the mixin here in case we can fix the translation issue. Refer to https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/42#issuecomment-672757911 for some suggestions.
```

I added a comment that summarized our discussion about the reason why `translate` doesn't work with PUG's mixins. https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/42#issuecomment-672757911